e2e/evm-evm/example/cmd/cmd.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package cmd
+
+import (
+	evmCLI "github.com/ChainSafe/chainbridge-core/chains/evm/cli"
+	"github.com/ChainSafe/chainbridge-core/config"
+	"github.com/ChainSafe/chainbridge-core/e2e/evm-evm/example/app"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var (
+	rootCMD = &cobra.Command{
```

Comment:
If we change that relayer isn't in docker-compose we would have example app inside core-example where I think it should belong and we wouldn't need the whole setup just for tests.

Reply:
I agree on this one with @mpetrun5 .

Reply:
i do not understand what you mean here

Reply:
It is mostly related to the discussion above about removing relayer from docker-compose. 
You wouldn't need the whole example app you would just run it directly and leave the testing 
of the example app to chainbridge-core-example repo.

Reply:
like cloning bulilding and running  chainbridge-core-example repo?

Reply:
Not really, the initial e2e idea was to trigger pipeline run in core-example repo.
Essentially, the e2e here would test just that transfers work and be something like a sanity run and when code is merged into main we would trigger an e2e run in core-example repo that would test it all.

Reply:
Still can't get it